### ISSUE TITLE
advertise jekyll-asciidoc

### ISSUE THREAD

**Issue by cup** [asciidoctor/asciidoctor.org#840]
as AsciiDoctor has a single-file scope, it might be useful to point users to
tools to help them deal with multiple files, like:

https://github.com/asciidoctor/jekyll-asciidoc

i have seen mentions of jekyll here and there, but it might be nice to link to
the plugin repo either on this repo, on https://asciidoctor.org or both.

**Comment by cup** [comment]
https://github.com/asciidoctor/asciidoctor/issues/2027

**Comment by mojavelinux** [comment]
I agree these should be mentioned on asciidoctor.org (or at least the upcoming docs.asciidoctor.org). It could go in this section https://asciidoctor.org/docs/#integrations-and-plugins, perhaps even on a dedicated page that lists integrations.

**Comment by mojavelinux** [comment]
I'd even broaden this a bit. I think the statement I made about the relationship between the Asciidoctor processor and build tools / static site generators needs to go in the docs. This will help users understand where everything fits.
